apps/api/src/app/change/e2e/promote-changes.e2e.ts

```diff
@@ -16,8 +17,9 @@ import {
 
 describe('Promote changes', () => {
   let session: UserSession;
+  const cacheService = new CacheService({ cacheHost: 'localhost', cachePort: '6379' });
```

Comment:
Shouldn't this be using the environment variables you set up?

Reply:
Totally right 😀 thanks for raising it up